inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -311,9 +327,17 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
 
     writeBits(header, os);
 
+    for (auto &name : inputNames) {
+        writeBits(static_cast<uint32_t>(name.size() + 1), os);
+        writeNBytes(name.c_str(), strlen(name.c_str()) + 1 , os);
+    }
     for (const auto &input : inputs) {
         writeBits(convert_to_serial(input), os);
     }
+    for (auto &name : outputNames) {
+        writeBits(static_cast<uint32_t>(name.size() + 1), os);
+        writeNBytes(name.c_str(), strlen(name.c_str()) + 1 , os);
```

Comment:
more consistent would be:
`const auto nameSize = strlen(name.c_str()) + 1;`
`writeBits(static_cast<uint32_t>(nameSize), os);`
`writeNBytes(name.c_str(), nameSize  , os);`